code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -294,22 +301,22 @@
 		return REACTING
 
 /datum/gas_reaction/nobeliumformation //Hyper-Nobelium formation is extrememly endothermic, but requires high temperatures to start. Due to its high mass, hyper-nobelium uses large amounts of nitrogen and tritium. BZ can be used as a catalyst to make it less endothermic.
-	priority = 3
+	priority = 6
 	name = "Hyper-Nobelium condensation"
 	id = "nobformation"
 
 /datum/gas_reaction/nobeliumformation/init_reqs()
 	min_requirements = list(
-		"nitrogen" = 10,
+		"n2" = 10,
 		"tritium" = 5,
-		"temp" = 500000)
+		"TEMP" = 500000)
 
 /datum/gas_reaction/nobeliumformation/react(datum/gas_mixture/air)
 	var/list/cached_gases = air.gases
 	air.assert_gases("nob","bz")
 	var/old_heat_capacity = air.heat_capacity()
 	var/nob_formed = (cached_gases["nitrogen"][MOLES]*cached_gases["tritium"][MOLES])/100
-	var/energy_taken = nob_formed*(1000000/cached_gases["bz"][MOLES])
+	var/energy_taken = nob_formed*(1000000/(cached_gases["bz"][MOLES])+1)
```

Comment:
are you trying to avoid a divby0 here? if so, you misplaced the close paren i think. should be `(1000000/(cached_gases["bz"][MOLES]+1))`